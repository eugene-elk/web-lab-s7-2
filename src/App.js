import React from 'react';
import MiniInfo from './components/MiniInfo'
import BigInfo from './components/BigInfo'

function App() {
  return (
    <div>
        <div class="header">
            <h1>Погода сейчас</h1>
        </div>
        <BigInfo />
    </div>
  );
}

export default App;
