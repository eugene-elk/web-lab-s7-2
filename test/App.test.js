import React from 'react';
import { shallow } from '../src/enzyme';
import { shallowToJson  } from 'enzyme-to-json';
import App  from '../src/App';

it('App отображается корректно', () => {
    const fullApp = shallow(
        <App />
    );
    expect(shallowToJson(fullApp)).toMatchSnapshot();
});
