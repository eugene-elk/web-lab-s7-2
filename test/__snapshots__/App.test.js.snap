// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App отображается корректно 1`] = `
<div>
  <div
    class="header"
  >
    <h1>
      Погода сейчас
    </h1>
  </div>
  <BigInfo />
  <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <PersistGate
      loading={null}
      persistor={
        Object {
          "dispatch": [Function],
          "flush": [Function],
          "getState": [Function],
          "pause": [Function],
          "persist": [Function],
          "purge": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
          Symbol(observable): [Function],
        }
      }
    >
      <Connect(FavoriteCityPack) />
    </PersistGate>
  </Provider>
</div>
`;
